integration_tests/__init__.py

```diff
@@ -366,6 +366,12 @@ def setUp(self):
         self.test_store = fixture_setup.TestStore()
         self.useFixture(self.test_store)
 
+    def is_store_fake(self):
+        return (os.getenv('TEST_STORE') or 'fake') == 'fake'
```

Comment:
This causes a bit of a double-take to determine that you actually want to default to the fake store. I'd prefer using a more explicit:

```python
def is_store_fake(self):
    test_store = os.getenv('TEST_STORE')
    if test_store and test_store != 'fake':
        return False
    return True
```

Reply:
Ah heck, nevermind. That's no more clear.